src/i18n/TextCollection/i18nTextCollector.php

```diff
@@ -622,10 +623,14 @@ public function collectFromCode($content, $fileName, Module $module)
             $previousToken = $thisToken;
             $thisToken = $token;
             if (is_array($token)) {
-                list($id, $text) = $token;
+                list($id, $text, $lineNo) = $token;
+                // minus 2 is intentional
```

Comment:
Intentional why? Please add more detail to the comment.

Reply:
Updated